bitnami/mariadb-galera/templates/secrets.yaml

```
@@ -13,3 +13,20 @@ data:
   ldap-bindpw: {{ .Values.ldap.bindpw | b64enc | quote }}
   {{- end }}
 {{- end }}
+{{- if .Values.ssl.enabled }}
+{{ if .Values.ssl.certificates }}
```

You can use the `and` condition here

```suggestion
{{- if and .Values.ssl.enabled .Values.ssl.certificates }}
```